### PR TITLE
Fixing issue #11046 for Columbia_University.xml 

### PR DESCRIPTION
**Pull request by HTTPSNowhere**
Take a look at the reasons given here: https://github.com/EFForg/https-everywhere/issues/11046

### CODE REVIEW

**Comment by J0WI** [comment]
This is definitely not a best practice ruleset, but to fix the issue you should just remove the rule for the `hr` domain and document the issue with a comment.

**Comment by HTTPSNowhere** [comment]
Thanks for the advice @J0WI, I initially wanted to audit the full ruleset but I didn't have the time. Anyway (I think) I uncovered `hr`.

**Comment by jeremyn** [comment]
`Columbia_University.xml` is in the testing whitelist [here](https://github.com/EFForg/https-everywhere/blob/92f0cbc9bfa20f1025eeba80926ce07965efad8c/utils/ruleset-whitelist.csv#L772), so if you just update the SHA256 hash there, that should fix the Travis error.

**Comment by HTTPSNowhere** [comment]
@jeremyn Done!

**Comment by HTTPSNowhere** [comment]
@jeremyn Just added it to the list, hope this fix the issue.

**Comment by jeremyn** [comment]
@HTTPSNowhere I checked out your branch and the SHA256 hash I'm getting for your modified `Columbia_University.xml` is `c2b1db224f8a0e740d3a13fd1f4259b7059cdf6b5fb3e5529ff35f0820068167`, not `d499ad33f006617f586a0e8c42675e4a4f5a6df515c2e81b1547f2d9b9f5d2e8` which is what you have.

**Comment by HTTPSNowhere** [comment]
@jeremyn Edited it again.

File: src/chrome/content/rules/Columbia_University.xml [hunk 1]
```diff
@@ -71,7 +71,7 @@
 	⁶ Times out
 	⁷ record_too_long
 	⁸ Shows columbiadoctors, mismatched, CN: columbiadoctors
-	⁹ Prints "Incorrect URL"
+	⁹ Prints 'Incorrect URL'
```
Comment (J0WI) [inline src/chrome/content/rules/Columbia_University.xml]: Please add `hr` to the list of nonfunctional subdomains.

**Comment by HTTPSNowhere** [comment]
Thanks @J0WI, it seems like it's all fixed now.

**Comment by J0WI** [comment]
You also need to update the checksum in the whitelist ;)

**Comment by HTTPSNowhere** [comment]
@J0WI Oh, right keep forgetting :P Fixed now!

**Comment by J0WI** [comment]
Thanks!

**Pull request merged**